dev/com.ibm.ws.javaee.ddmodel/resources/com/ibm/ws/javaee/internal/ddmodel/resources/DDModelMessages.nlsprops

```diff
@@ -110,13 +196,23 @@ found.duplicate.attribute.value=CWWKC2270E: The {2} attribute for all {1} elemen
+
+# REPLACED BY:
 # -----------------------------------------------------------------------
 # PARAMETERS:
-#    {0} is the path name for the module
+#    {0} is the path to a validation configuration file, including paths to parent archives
+#    {1} is the module name
+#    {2} is the number of a line within the configuration file
+#    {3} is the actual root element name
+#    {4} is the expected root element name
 # -----------------------------------------------------------------------
-BVKEY_NOT_A_BEAN_VALIDATION_XML=CWWKC2271W: A validation.xml file was found for the {0} module. However, that validation.xml file is not configured for validation; therefore this XML file is ignored.
-BVKEY_NOT_A_BEAN_VALIDATION_XML.explanation=The BeanValidationService cannot create a ValidatorFactory because of an error parsing the validation.xml file with the validation-configuration-1.0.xsd schema file. This error may indicate that the validation.xml file is not intended for use by the Bean Validation run-time; therefore, this XML file is ignored.
-BVKEY_NOT_A_BEAN_VALIDATION_XML.useraction=If the validation.xml file is not for creating a ValidatorFactory instance, then ignore the message. However if the validation.xml file is intended to create a ValidatorFactory instance, then a problem exists.  Enable Bean Validation run-time trace in the application server, stop the application, restart the application, and examine the trace output file for a trace event such as: Unexpected exception when trying to unmarshall the validation.xml file. Examine the exception stack trace for debugging assistance.
+unexpected.root.element.bval=CWWKC2271W: Ignoring bean validation configuration file {0} of module {1}.  At line number {2}, the root element is {3} but should be {4}.
+unexpected.root.element.bval.explanation=The validation configuration file does not match the validation configuration schemas (validation-configuration-1.0.xsd and validation-configuration-1.1.xsd). Either the configuration file was not intended for use for bean validation, or the configuration file is malformed.  The configuration file is being ignored.  The BeanValidationService will not create a ValidatorFactory.
+unexpected.root.element.bval.useraction=If the validation configuration file is not intended for bean validation, the warning message can be ignored.  If the validation configuration file is intended for bean validation, the warning message should not be ignored, as the bean validation service is not running as intended.  To diagnose the problem, enable bean validation runtime trace, stop and restart the server, and examine trace output for unexpected exceptions relating to bean validation. For example, "Unexpected exception when trying to unmarshall the validation.xml file."  Examine the exception stack trace for debugging assistance.
```

Comment:
Change "the warning message should not be ignored, as" to "do not ignore the warning message because"